packages/create-universal-app/bin/generate.js

```diff
@@ -3,6 +3,7 @@ import { spawn } from "child_process";
 import fs from "fs-extra";
 import ora from "ora";
 import path from "path";
+import { getUserInput } from "../utils/getUserInput.js";
 
 const repoURL = "https://github.com/theodo-group/universal-app.git";
```

Comment:
not your PR but I think constants should following the `CAPITAL_LETTERS` format usually found